ccdproc/image_collection.py

```diff
@@ -58,14 +58,18 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     ext: str or int, optional
+         The extension from which the header and data will be read in all files.
```

Comment:
Let's leave it in to emphasize that the data will be read from the same extension for all of the files.